.github/workflows/devbuild.yml

```diff
@@ -249,4 +263,12 @@ jobs:
             -Dpybind11_DIR="$(pybind11-config --cmakedir)" `
             -S${{ github.workspace }} `
             -B${{ github.workspace }}/build
-          cmake --build ${{ github.workspace }}/build --config ${{ matrix.cmake_build_type }} --target ALL_BUILD
\ No newline at end of file
+          cmake --build ${{ github.workspace }}/build `
+            --config ${{ matrix.cmake_build_type }} `
+            --target ALL_BUILD
+
+      - name: cmake run_viewer_pytest
+        run: |
+          cmake --build ${{ github.workspace }}/build `
+            --config ${{ matrix.cmake_build_type }} `
+            --target run_viewer_pytest
```

Comment:
Unit testing on Windows is now running, but the return code is not checked yet.

There is a minor error that can be fixed in the future when we change to respect the return code.